src/components/typography/H2.tsx

```diff
@@ -1,45 +1,29 @@
-import React from "react";
-import {
-  IOColors,
-  IOColorsStatusForeground,
-  IOTheme
-} from "../../core/IOColors";
+import { IOTheme } from "../../core/IOColors";
 import { FontFamily, IOFontWeight } from "../../utils/fonts";
 import { useTypographyFactory } from "./Factory";
 import { ExternalTypographyProps, TypographyProps } from "./common";
 
-type PartialAllowedColors = Extract<
-  IOColors,
-  "bluegreyDark" | "white" | "blue" | "bluegrey"
->;
-type AllowedColors =
-  | PartialAllowedColors
-  | IOColorsStatusForeground
-  | IOTheme["textHeading-default"];
-type AllowedWeight = Extract<IOFontWeight, "Bold" | "SemiBold">;
+type AllowedColors = IOTheme["textHeading-default"];
+type AllowedWeight = Extract<IOFontWeight, "SemiBold" | "Regular">;
```

Comment:
```suggestion
type AllowedWeight = Extract<IOFontWeight, "Regular">;
```

Reply:
@CrisTofani If we just keep the new style (and it's a big IF 😅), there is only one weight: `Regular`, because all titles (except `H5`) use `ReadexPro`, which comes with only one weight.